diff -ru vice-3.5/src/arch/sdl/icon.c vice-3.5-patched/src/arch/sdl/icon.c
--- vice-3.5/src/arch/sdl/icon.c	2020-10-07 09:56:38.000000000 -0500
+++ vice-3.5-patched/src/arch/sdl/icon.c	2020-12-24 14:18:00.786630036 -0600
@@ -28,7 +28,6 @@
 
 #include "archdep.h"
 #include "vice_sdl.h"
-#include <SDL_image.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdint.h>
@@ -40,6 +39,8 @@
 
 #ifdef USE_SDLUI2
 
+#include <SDL2/SDL_image.h>
+
 void sdl_ui_set_window_icon(SDL_Window *window)
 {
     SDL_Surface *surface;
